code/modules/supply/supply_packs/pack_engineering.dm

```diff
@@ -220,3 +220,45 @@
 	containername = "thermo-electric generator crate"
 	access = ACCESS_CE
 	announce_beacons = list("Engineering" = list("Chief Engineer's Desk", "Atmospherics"))
+
+/datum/supply_packs/engineering/canister/nitrogen
+	name = "Nitrogen canister"
+	contains = list(/obj/machinery/atmospherics/portable/canister/nitrogen)
+	cost = 50
+	containertype = /obj/structure/largecrate
+	containername = "nitrogen canister crate"
+
```

Comment:
also you should probably make sure these are department restricted to engineering if they're not already

Reply:
As far as I'm aware, it being a subtype of /supply_packs/engineering/ already puts it into the department restriction, but I could be very wrong here